l2geth/consensus/clique/synchronizer/server.go

```diff
@@ -88,9 +98,20 @@ func (sync *Synchronizer) GetSequencerSet() (SequencerSequencerInfos, error) {
 	}
 	// get sequencers from sequencer contract
 	var seqInfos SequencerSequencerInfos
-	seqInfos, err = seqContractInst.GetSequencers(nil)
-	if err != nil {
-		return nil, err
+	for i := 0; i < GET_SEQUENCER_RETRY_TIMES; i++ {
+		seqInfos, err = seqContractInst.GetSequencers(nil)
+		if err != nil {
+			return nil, err
```

Comment:
if an error happens, the retry will be aborted.

Reply:
> if an error happens, the retry will be aborted.

yeah, replace with continue.